docs/targets/googlesheets.md

```diff
@@ -1,42 +1,43 @@
 # Event Target for Google Sheets
 
-This event target receives [CloudEvents][ce] over HTTP and appends the event payload to a GoogleSheets sheet.
+This event Target receives [CloudEvents][ce] over HTTP and appends the event payload to a GoogleSheets Sheet.
 
-## Prerequisites
+## Prerequisite(s)
 
-1. Google API Credentials
-1. GoogleSheets Sheet id
+- Google API Credentials
+- GoogleSheets Sheet ID
 
-### Google API Credentials
+Consult the [Secrets](../guides/secrets.md) guide for more information about
```

Comment:
Should this be moved to the section about API creds?

Reply:
I am open to moving it there. I initially chose to put it with the prerequisites since the other pages have this in the prerequisites section.

Reply:
I think I'll go ahead and move it after thinking about it a bit more. It would be more relevant with the credentials.